src/include/duckdb/storage/statistics/base_statistics.hpp

```diff
@@ -104,6 +104,7 @@ class BaseStatistics {
 	static BaseStatistics Deserialize(Deserializer &deserializer);
 
 	//! Verify that a vector does not violate the statistics
+	void Verify(Vector &vector, const SelectionVector &sel, idx_t count, bool ignore_has_null) const;
```

Comment:
nit: `const bool ignore_has_null = false` and only pass `true` for `StructStats::Verify`.